src/coreclr/debug/runtimeinfo/datadescriptor.h

```diff
@@ -152,6 +152,18 @@ CDAC_TYPE_FIELD(PreviousNestedInfo, /*pointer*/, PreviousNestedInfo, offsetof(Ex
 #endif
 CDAC_TYPE_END(ExceptionInfo)
 
+CDAC_TYPE_BEGIN(ExceptionObject)
```

Comment:
I think that this is first instance where we have a data descriptor for a managed type. It is fine for now. I am wondering whether we will want to have a different infrastructure at some point in future that allows describing data descriptor of managed types without mirroring them into unmanaged runtime.

Reply:
Contracts for managed types is going to be required to support Native AOT.  Unless I'm mistaken the Exception object isn't mirrored in the C++ side.